corehq/sql_accessors/migrations/0018_soft_delete_cases.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+from corehq.sql_db.operations import RawSQLMigration, HqRunSQL
+
+migrator = RawSQLMigration(('corehq', 'sql_accessors', 'sql_templates'))
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sql_accessors', '0017_merge'),
+    ]
+
+    operations = [
+        HqRunSQL(
+            "DROP FUNCTION IF EXISTS soft_delete_cases(TEXT, TEXT[], TIMESTAMP, TEXT)",
+            "SELECT 1"
```

Comment:
@snopoke i copied this migration and the other one from the previous soft_delete_cases ddl migration. have a look and let me know if everything looks ok.  just for my own understanding, how come we have to select 1 here?